packages/api/src/app-router.ts

```diff
@@ -254,12 +254,16 @@ export default async function makeApp(params: CliArgs) {
   }
   const apiErrorHandler = errorHandler();
 
-  app.use((err, _req, res, _next) => {
-    if (frontend && !_req.path.startsWith(httpPrefix)) {
-      wwwHandler(err, _req, res, _next);
-    }
-    apiErrorHandler(err, _req, res, _next);
-  });
+  if (frontend) {
+    app.use((req, res, next) => {
```

Comment:
Hmmmm, maybe in the end what we really wanted here was doing an `app.all('*', ...)` which registers a catch-all fallback route instead of `app.use` which registers a middleware?